fixtures/services/bulk-item-pickup.yaml

```diff
@@ -111,6 +112,10 @@ additional_content_ar: |
         <li>السيارات المركونة.</li>
     </ul>
 
+dynamic_content:
+    -
+        type: collection_schedule
```

Comment:
the type previously was collection_schedule_block.
We'll need to update the switch statement in src/js/modules/ApplicationBlock

Reply:
This won't change the API response